src/components/verification-grid-tile/css/style.css

```diff
@@ -15,9 +27,11 @@
   box-shadow: var(--oe-backdrop-shadow) var(--oe-panel-color);
   cursor: pointer;
   margin-bottom: 0;
-  min-width: 350px;
+  min-width: 300px;
+  min-height: 300px;
```

Comment:
is this tied to the minimum height in our grid calculations? comment and cross link at a minimum